_data/converters.yml

```diff
@@ -231,3 +231,11 @@
       documentation: https://exploreasl.github.io/Documentation/1.8.0/Tutorials-ASL-BIDS/
       language: MATLAB
       updated: 2021-09-13
+
+    - name: SAMRI
+      url: https://github.com/IBT-FMI/SAMRI
+      updated: 2021-11-08
+      data_types: small-animal MRI
+      language: Python, shell
+      documentation: https://doi.org/10.3389/fninf.2020.00005
+      GUI: false
```

Comment:
@TheChymera this PR adds the SAMRI converter to the list of bids compatible data converters/tools (see https://bids.neuroimaging.io/benefits.html#mri-and-pet-converters).

I filled in the info as well as I could from a brief glance --> could you please (by looking at the other entries in this YML file) add information that I missed, and/or correct what I got wrong?

Reply:
@sappelhoff thank you! I could help out with this, but I can't really understand where/why the markdown link check fails :-/

Reply:
I fixed it by removing it :) ... was some link in an old blog post that nobody is going to miss.

Reply:
but your help would be appreciated by either approving of the changes ... or telling me which additional info I should include.

Reply:
fair point, I added it back and changed the link to be a proper `doi.org` link instead of the `sagepub` link. Although it might just redirect there anyways.

Reply:
Failure may be related to https://github.com/tcort/markdown-link-check/issues/109.

Reply:
should I just write a comment here for what's to be added, or submit a PR to your branch? I think it might be easiest if I just submit a PR for additional info after this is merged.

Reply:
@TheChymera sounds good - looking forward to your follow-up PR to improve the information!

Reply:
https://github.com/bids-standard/bids-website/pull/224 for the follow-up :)